packages/components/src/theme.test.ts

```diff
@@ -0,0 +1,60 @@
+import { render } from '@testing-library/react';
+import { h } from 'react-hyperscript-helpers';
+
+import { ErrorBoundary } from './ErrorBoundary';
+import { Theme, ThemeProvider, useThemeFromContext } from './theme';
+
+describe('useContextFromTheme', () => {
```

Comment:
um... this name seems fubbled.

Reply:
How so?

Reply:
Fixed in https://github.com/DataBiosphere/terra-ui/pull/4131/commits/c84fb6938da206e64fcf84f2143c18a7a9cd1b3d.